content/en/contribute/medic/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "At Medic"
+linkTitle: "At Medic"
+weight: 4
+description: >
+  Compilation of guidelines and product development processes used at Medic 
+---
+
+The Product team works with Medic teammates, partners, and health workers to design, build, and maintain several open source tools. We work with our partners along the entire software development life-cycle to understand problems, capture requirements, design and build modular software systems, and document everything along the way.
```

Comment:
Reframing for more general audience.


```suggestion
[Medic](https://medic.org) works with CHT partners and health workers to design, build, and maintain the Community Health Toolkit and its open source tools. Medic's Product team manages the entire software development life-cycle to understand problems, capture requirements, design and build modular software systems, and document everything along the way.
```

Reply:
The ["no we"](https://docs.communityhealthtoolkit.org/contribute/docs/style-guide/#using-we) rule will help to get the tone right for the new public home of this content.